subcommand/server-acl-init/command.go

```diff
@@ -73,7 +74,7 @@ type Command struct {
 func (c *Command) init() {
 	c.flags = flag.NewFlagSet("", flag.ContinueOnError)
 	c.flags.StringVar(&c.flagResourcePrefix, "resource-prefix", "",
-		"Prefix to use for Kubernetes resources. If not set, the \"<release-name>-consul\" prefix is used, where <release-name> is the value set by the -release-name flag.")
```

Comment:
This was missed when we removed the `-release-name` flag.

Reply:
🙏